src/io/jesi/backpack/string.cljc

```diff
@@ -134,3 +134,17 @@
   (some-> s
           (str/replace \- \space)
           (->proper-case)))
+
+(defn split-at-first
+  "Splits s at the first occurrence of value"
+  [value s]
+  (let [value (str value)]
+    (cond
+      (empty? s) []
```

Comment:
Or?
```suggestion
      (empty? s) nil
```
Not sure what our convention should be. I checked https://github.com/jesims/docs/blob/master/Coding-Conventions/Clojure.md and it's links.
I'm leaning towards `nil`

Reply:
I agree with `nil` it would make it's usage easier than having to do an empty check on a vector (e.g. when using it with `(when-let)`